wemake_python_styleguide/visitors/ast/function_empty_lines.py

```diff
@@ -0,0 +1,65 @@
+import ast
+import math
+from contextlib import suppress
+from copy import copy
+
+from typing_extensions import final
+
+from wemake_python_styleguide import types
+from wemake_python_styleguide.violations import best_practices
+from wemake_python_styleguide.visitors import base
+from wemake_python_styleguide.visitors.decorators import alias
+
+
+@final
+@alias('visit_any_function', (
+    'visit_FunctionDef',
+    'visit_AsyncFunctionDef',
+))
+class WrongEmptyLinesCountVisitor(base.BaseNodeVisitor):
+    """Restricts empty lines in function or method body."""
+
+    def visit_any_function(self, node: types.AnyFunctionDef) -> None:
+        """Find empty lines count."""
+        start_line = node.lineno
+        if not start_line or not node.end_lineno:
+            return
+        lines_range = set(range(start_line, node.end_lineno))
+        lines_without_expressions = self._lines_without_expressions(
+            copy(lines_range), node,
```

Comment:
NIP: `set` has `copy` method that can be used to create shallow copy of a `set` instance:

```suggestion
            lines_range.copy(), node,
```

Reply:
Didn't know about this method